rusk-wallet/src/bin/interactive/command_menu.rs

```diff
@@ -169,14 +169,21 @@ pub(crate) async fn online(
                 .public_key(stake_idx)
                 .expect("public key to exists in interactive mode");
 
+            let mut has_stake = false;
+
             let owner = match wallet.find_stake_owner_account(stake_pk).await {
-                Ok(account) => account,
+                Ok(account) => {
```

Comment:
Having a stake owner is not enough to know if there is a stake amount

Indeed, if the user has no stake but still has rewards to withdraw, this match will incorrectly results in "has_stake=true"

Reply:
@herr-seppia I fixed this check please check again